.github/pull_request_template.md

```diff
@@ -1,49 +1,11 @@
 ---
 name: Pull Request Template
 about: Default pull request template
-title: ''
+title: '<type>[optional scope]: <description>'
```

Comment:
Shall we add list of types like it was before ( bug fix, new feature, breaking change, documentation only).
What is scope here? FE, BE, test, devops or something else?

Reply:
I can add some support documentation, following the example of Angular: https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type